app/Services/VersionService.php

```diff
@@ -91,13 +95,13 @@ private function getPossibleVersionResponse(Version $userVersion, Version $lates
      */
     public function getVersion(string $versionString) : Version
     {
-        $version = Version::where('version', '=', $versionString);
+        $version = Version::where('version', '=', $versionString)->withTrashed()->first();
```

Comment:
Same here, you could catch ModelNotFound and just have a return in the try / catch